server/routes/api/representatives.js

```diff
@@ -129,6 +130,72 @@ router.get('/:zipCode', async (req, res) => {
   }
 })
 
+/*
+ * Returns string with the following properties:
+ *  - x-value: right or left. Indicates whether the photo should be cropped/positioned from the left or right side.
+ *  - y-value: top or bottom. Indicates whether the photo should be cropped/psoitioned from the top or bottom side.
+ *
+ * Example: "top left"
+ *
+ * @param {string} photoCropping - The photo cropping object from the CICERO API
+ * @returns {string} - The photo cropping CSS value
+ *
+ *
+ */
+function getPhotoCroppingValues(photo_cropping_object) {
+  // If the photo cropping object is not defined, return default value
+  if (!photo_cropping_object) {
+    return 'center center'
+  }
+
+  // args
+  console.log(photo_cropping_object)
+  let x = photo_cropping_object.x
+  let y = photo_cropping_object.y
+  let oriH = photo_cropping_object.oriHeight
+  let oriW = photo_cropping_object.origWidth
+
+  // 1. calculate threeshold for the x space
+  // we check if the coordinate starts on the left side of the image (the first half of the left side)
+  let x_left_threeshold = oriW / 2 / 2
+  // we check if the coordinate starts on the right side of the image (the first half of the right side) and we reduce a margin of 5% to be flexible
+  let x_right_threeshold = oriW / 2 - (5 / 100) * (oriW / 2)
+  console.log('x_left_threeshold: ' + x_left_threeshold)
+  console.log('x_right_threeshold: ' + x_right_threeshold)
+  // 2. calculate threeshold for the y space
+  // we check if the coordinate starts on the top side of the image (the first half of the top side)
+  let y_top_threeshold = oriH / 2 / 2
+  // we check if the coordinate starts on the bottom side of the image (the first half of the bottom side) and we reduce a margin of 5% to be flexible
+  let y_bottom_threeshold = oriH / 2 - (5 / 100) * (oriH / 2)
+  console.log('y_top_threeshold: ' + y_top_threeshold)
+  console.log('y_bottom_threeshold: ' + y_bottom_threeshold)
+
+  // 3. determine the css values for the cropping
+  let x_value =
+    x <= x_left_threeshold
+      ? 'left'
+      : x >= x_right_threeshold
+      ? 'right'
+      : 'center'
+  let y_value =
+    y <= y_top_threeshold
+      ? 'top'
+      : y >= y_bottom_threeshold
+      ? 'bottom'
+      : 'center'
```

Comment:
To improve readability, don't chain the ternary operations together.  I would suggest assigning the second part of each ternary operation to its own variable, and then put the resulting value into the assignments here.

Reply:
I am not sure if I understand the suggestion. I understand the "if-else-if-else" block can cause confusion, but it is hard for me to picture the second part being alone (the else-if block) as another ternary operation. Nevertheless, I will do my best to improve readability.

Reply:
I have used if-else blocks to make the code easier to read.

Reply:
Sorry, I should have been clearer.  I just meant something like:

```js
  let x_right = x >= x_right_threshold ? 'right' : 'center'
  let x_value = x <= x_left_threshold ? 'left' : x_right
```

I think the code you replaced it with is also more readable.  Up to you if you want to switch back to using the ternary pattern, thanks for considering readability anyways!

Reply:
I agree this alternative is more readable, and I tried to prevent it from being verbose. Thanks. I will keep this version.